app/javascript/components/ItemCard.jsx

```diff
@@ -3,9 +3,57 @@ import Placeholder1 from "images/item_placeholder_1.JPG";
 import Placeholder2 from "images/item_placeholder_2.JPG";
 import EditIcon from "images/icons/edit_icon.svg";
 import DeleteIcon from "images/icons/delete_icon.svg";
+import { useCookies } from "react-cookie";
+
 
 export default(props) => {
 	const [onHover, setOnHover] = useState(false);
+	const [errorMessage, setErrorMessage] = useState(null);
```

Comment:
Do we have plans to use / display this error message?

Reply:
I don't think we do, it's just something I included incase we did want to/for consistency sake.